core/new-gui/src/app/common/type/workflow.ts

```diff
@@ -14,14 +14,13 @@ import { Breakpoint, OperatorLink, OperatorPredicate, Point } from '../../worksp
  *  will then be used to reload the entire workflow.
  *
  */
-export interface WorkflowContent {
+export interface WorkflowContent extends Readonly<{
   operators: OperatorPredicate[];
   operatorPositions: { [key: string]: Point };
+  disabledOperators: string[];
```

Comment:
I am worried about the compatibility of this code. Remember we have workflows saved as JSON strings in prod DB. Changing the frontend representation of it in my understanding would read the `disabledOperators` as undefined. Would that cause some unexpected issues?

Reply:
Another design might be to add a property on each `OperatorPredicate`, namely `disable: boolean`. In this way, we could do the same for `OperatorLink` in the future as well. 

what do you think?

Reply:
changed `disabledOperators` to be optional so it's backwards compatible

Reply:
Add a property to `OperatorPredicate` sounds a good idea, we can do the factor later because I don't want to spend too much time on this task,

Reply:
changed the implementation to add an optional property in `OperatorPredicate`